docs/config-file/v2.rst

```diff
@@ -323,6 +326,32 @@ as defined here:
 * `stable <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/5.x>`_: :buildpyversions:`stable`
 * `latest <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/6.x>`_: :buildpyversions:`latest`
 
+build.apt_packages
+``````````````````
+
+List of `APT packages`_ to install.
+Our build servers run Ubuntu 18.04, with the default set of package repositories installed.
```

Comment:
IIRC, `testing` runs 20.04. It may worth to mention this somehow as well.

Reply:
Looks like testing is still running on 18.04 https://github.com/readthedocs/readthedocs.org/issues/8151#issuecomment-831440652